src/entity.cpp

```diff
@@ -24,3 +22,34 @@ void Entity::update(float frameTime) {
 }
 
 void Entity::ai(float frameTime, Entity &ent) {}
+
+void Entity::rotateEntity(std::string direction) {
+	RECT sampleRect = this->getSpriteDataRect();
+
+	if (direction != "") {
+		sampleRect.left = 0;
+		sampleRect.right = TILE_SIZE;
+
+		if (direction == "LEFT")
+			sampleRect.top = 64;
+		if (direction == "RIGHT")
+			sampleRect.top = 96;
+		if (direction == "UP")
+			sampleRect.top = 32;
+		if (direction == "DOWN")
+			sampleRect.top = 0;
+
+		sampleRect.bottom = sampleRect.top + TILE_SIZE;
+	}
+
+	this->setSpriteDataRect(sampleRect);
+}
+
+void Entity::setRect() {
+	spriteData.rect.left = currentFrame * spriteData.width;
+	spriteData.rect.right = spriteData.rect.left + spriteData.width;
+}
+
+void Entity::startWalkAnimation() {}
+
+void Entity::startAttackAnimation() {}
```

Comment:
Not sure if you require these 2 functions in Entity class since there isnt any default code for it